eng/build.yml

```diff
@@ -15,10 +15,11 @@ parameters:
   timeoutInMinutes: 180
 
 jobs:
-- ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
+- ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest'), eq(variables['Build.SourceBranch'], 'refs/heads/release/5.0.4xx')) }}:
```

Comment:
@sfoslund Do we need this?

Reply:
No, we don't need any of the changes in this file.